logstash-plugins/usagetype-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
Could distribition/zipStore Base/Path be consolidated to one variable each?

Reply:
Maybe. I tried not to touch the gradle stuff any more than was necessary. At least not until I understand it better.